security/pkg/stsservice/server/server.go

```diff
@@ -81,8 +82,10 @@ func NewServer(config Config, tokenManager security.TokenManager) (*Server, erro
 	mux.HandleFunc(TokenPath, s.ServeStsRequests)
 	mux.HandleFunc(StsStatusPath, s.DumpStsStatus)
 	s.stsServer = &http.Server{
-		Addr:    fmt.Sprintf("%s:%d", config.LocalHostAddr, config.LocalPort),
-		Handler: mux,
+		Addr:        fmt.Sprintf("%s:%d", config.LocalHostAddr, config.LocalPort),
+		Handler:     mux,
+		IdleTimeout: 90 * time.Second, // matches http.DefaultTransport keep-alive timeout
```

Comment:
Does it make sense for the 90/30 to come from config.IdleTimeout/config.ReadTimeout vs having it baked into the compiled code?

Reply:
@ryan-dayrit What do you mean?

Reply:
- Extending the config struct to have these IdleTimeout / ReadTimeout props. 
- Setting default vals to 90 / 30 respectively 

```
IdleTimeout: config.IdleTimeoutSecs * time.Second
ReadTimeout: config.ReadTimeoutSecs * time.Second
```